packages/cozy-konnector-libs/src/libs/saveBankingDocuments.js

```diff
@@ -1,36 +1,40 @@
 /**
- * Combines the features of `saveFiles`, `filterData`, `addData` and `linkBankOperations` for a
- * common case: bills.
- * Will create `io.cozy.bills` objects. The default deduplication keys are `['date', 'amount', 'vendor']`.
- * You need the full permission on `io.cozy.bills`, full permission on `io.cozy.files` and also
- * full permission on `io.cozy.bank.operations` in your manifest, to be able to * use this function.
+ * Combines the features of `saveFiles`, `filterData`, `addData` and `linkBankOperations` for any
+ * doctype.
+ * Will create `io.cozy.bills` documents by default or any specified doctype.
+ * The default deduplication keys are `['date', 'amount', 'vendor']`.
+ * You need the full permission on the doctype, full permission on `io.cozy.files` and also
+ * full permission on `io.cozy.bank.operations` in your manifest, to be able to use this function.
```

Comment:
*... to be able to use this function (unless option `noBanking: true` is set, see below).*?

Or will the permission still be needed anyway?

Reply:
That is right, I added the noBanking option later.

Then the name of the function should be saveDocuments since we may want to use it without banking, and since the banking part may be extracted from the connector anyway.